test_container/tests/test/r/scalar_returns_return_behavior.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python3
+
+from exasol_python_test_framework import udf
+
+
+class REmptyUDF(udf.TestCase):
+    def setUp(self):
+        self.schema = "REmptyUDF"
+        self.query(f'DROP SCHEMA {self.schema} CASCADE', ignore_errors=True)
+        self.query(f'CREATE SCHEMA {self.schema}')
+
+    def test_return_int_scalar_returns(self):
+        self.query(udf.fixindent('''
+                CREATE r SCALAR SCRIPT return_int_scalar_returns(i int)
+                RETURNS int AS
+
+                run <- function(ctx) {
+                    return(10)
+                }
+                /
+                '''))
+        rows = self.query('select return_int_scalar_returns(i) from VALUES BETWEEN 1 AND 9 AS t(i)')
+        self.assertRowsEqual([(10,) for i in range(1,10)], rows)
+
+    def test_return_vector_int_scalar_returns(self):
+        self.query(udf.fixindent('''
+                CREATE r SCALAR SCRIPT return_vector_int_scalar_returns(i int)
+                RETURNS int AS
+
+                run <- function(ctx) {
+                    return(c(10))
+                }
+                /
+                '''))
+        rows = self.query('select return_vector_int_scalar_returns(i) from VALUES BETWEEN 1 AND 9 AS t(i)')
+        self.assertRowsEqual([(10,) for i in range(1,10)], rows)
```

Comment:
This expects the same result as for the first tests case `test_return_int_scalar_returns`. Is this intended?

Reply:
Yes, this is intended, it is actually the means of this test